12 Character Password update

simon-leech opened the pull request:
Updated the default login and register views to use a minimum length of 12 characters.

cityremade reviewed (approved):
I see 10s changed to 12s 👍

dbauszus-glx reviewed (changes requested):
This does not work at the moment.

The information in regards to the length is shown as a tooltip which I won't see on a touchscreen.

The red light / green light indicator is invisible to people with a colour deficiancy. https://github.com/GEOLYTIX/xyz/issues/1764

The info text must emphasise that the password must be reset if the current password does not match the complexity schema.

![image](https://github.com/user-attachments/assets/d22887ef-9e01-41ff-838e-037032da0279)

We need to create a login template which uses a dictionary endpoint in the future. Having to change multiple language view templates for login and registration is unsustainable. I will create a ticket for this.

simon-leech commented:
Updated the login page: 
1. Added message about the password change
2. Added error message when email or password are invalid to prevent issues with accessibility. 
![image](https://github.com/user-attachments/assets/ced235c9-0f2a-44cd-8673-eba94fd64c88)


Please could this be reviewed before I do the same change on all the non-english login pages too?

RobAndrewHurst commented:
What happens to the message in a months time? Do we have to create another PR to remove it?

simon-leech commented:
> What happens to the message in a months time? Do we have to create another PR to remove it?

I would guess so yeah - I'm not sure how else to handle this really 😅

RobAndrewHurst commented:
![image](https://github.com/user-attachments/assets/f009c38f-0ed2-4372-83a0-4306e546ff91)

The Choose language label is quite far away from the drop down. I might want to be closer to the drop down itself.

simon-leech commented:
@RobAndrewHurst Good point - I've moved it next to it 👍 
![image](https://github.com/user-attachments/assets/ba28df91-74e7-4408-91e6-a32c11766cc4)

RobAndrewHurst reviewed (approved):
LGTM!

dbauszus-glx reviewed (approved):
I am happy for this to be merged. There are a couple of tweaks but I want to look at these after we are able to load the dictionary into a single login/registration view template.
